<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模拟测试工具V1.0 作者：shiyu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>不知道为什么</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>85</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>第一步：考试科目</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_subject">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>驾驶员</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>机长</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>145</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>第二步：考生信息</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_MainWindow_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>175</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>205</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>身份证号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_MainWindow_number">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>第三步：考试类型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_type">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>255</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>练习</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>考试</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>300</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>选择题库：(注意：路径中不能有中文)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_MainWindow_Path">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>315</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_MainWindow_open">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>315</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开题库文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_MainWindow_Start">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始考试</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>70</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>考试题量：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_MainWindow_TitleNumber">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>70</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>题库总数量：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_MainWindow_TitleNumber_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_MainWindow_Import">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导入题库</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>只是想写个标题。。。</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
